docs/manual/build_system/cmake_variables.qbk

```diff
@@ -38,14 +38,16 @@ The options are split into these categories:
 * [link build_system.cmake_variables.HPX_NATIVE_TLS HPX_NATIVE_TLS]
```

Comment:
It looks like not all cmake variables have been updated in the docs here. Could you rerun the doc generation please?